### ISSUE TITLE
"Aborting replay due to bad decode" for protocol81102

### ISSUE THREAD

**Issue by StanczakDominik** [ZephyrBlu/zephyrus-sc2-parser#4]
I'm getting errors on parsing my current replays:

```python
>>> zephyrus_sc2_parser.parse_replay("Deathaura LE (46).SC2Replay, local=True)
ERROR:root:An error occurred while decoding: No module named 'protocol81102'
ERROR:root:An error occurred while decoding: No module named 'protocol81102'
ERROR:root:An error occurred while decoding: No module named 'protocol81102'
ERROR:root:An error occurred while decoding: No module named 'protocol81102'
ERROR:root:An error occurred while decoding: No module named 'protocol81102'
CRITICAL:root:Replay could not be decoded
CRITICAL:root:Aborting replay due to bad decode...
```

with a replay such as (don't judge for mishandling the mine drop...) https://drive.google.com/file/d/13JlI6GqIEcwh6zj5X7jdv0TVrehcyXT_/view?usp=sharing

The latest s2protocol here is 81009, I guess.

I kinda wonder whether it might not be possible to automate the updates through a script, maybe even through CI? Perhaps I could help with that somehow.

**Comment by ZephyrBlu** [comment]
Thanks for the heads up. I think I'm a patch behind the game client (81102 must be 5.2). I'll be updating the parser within the next day or so :).

> I kinda wonder whether it might not be possible to automate the updates through a script, maybe even through CI? Perhaps I could help with that somehow

I would do that, but I can't predict what the protocol will be in advance.

**Comment by StanczakDominik** [comment]
Sure, what I meant is: is it possible to, given information that there is a new version of s2protocol, generate the necessary updates here somewhat automatically? What kind of steps do you usually need to take to do that?

**Comment by ZephyrBlu** [comment]
Not sure if I can automate that. What I usually do is copy the most recent protocol in this repo and just rename the file as the latest protocol, since Blizzard's s2protocol repo isn't updated that often. Most of the time this works fine. I also have to generate or copy version data for the latest patch though.

Maybe I need to look into GitHub's webhooks or something similar, but it feel like more work to automate the process than to just do it when it's needed. The only issue is sometimes I forget to update the parser haha.

**Comment by StanczakDominik** [comment]
Fixed in v0.2.25 :)
